Sources/NIOExtras/DebugOutboundEventsHandler.swift

```diff
@@ -0,0 +1,88 @@
+import NIO
+
+public class DebugOutboundEventsHandler: ChannelOutboundHandler {
```

Comment:
This class needs a doc comment.

Reply:
Sorry, but doc comments begin each line with the four character string: `/// ` (that is, three slashes and then a space).

Reply:
Ah right, sorry for not paying too much attention. Updated with real docs